src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/Deploy.kt

```diff
@@ -34,14 +35,18 @@ open class Deploy : Sync() {
      */
     @Internal
     @get:JsonIgnore
-    var uploadRetry = aem.retry { afterSquaredSecond(aem.props.long("aem.deploy.uploadRetry") ?: 6) }
+    var uploadRetry = aem.retry {
```

Comment:
why? I prefer inlined to be able to read it is a sentence ;)

Reply:
IDE formatting